src/main/menu_main.cpp

```diff
@@ -1812,7 +1812,7 @@ static void s_drawGameSaves(int MenuX, int MenuY)
     const auto& info = SaveSlotInfo[MenuCursor + 1];
 
     int infobox_x = ScreenW / 2 - 240;
-    int infobox_y = MenuY + 145;
+    int infobox_y = MenuY + 145 + c_menuSavesOffsetY;
 
```

Comment:
This usage of c_menuSavesOffset here suggests that you're centering the menu on its old coordinates, not shifting it up. Make sure that the info box stays in the same location when entering the 1P char select screen.

Reply:
I added this because I done the shift of the whole menu block, and I keeping info box being sticked to the bottom of the shifted menu.